docs/source/python/api/compute.rst

```diff
@@ -59,6 +59,28 @@ throws an ``ArrowInvalid`` exception when overflow is detected.
    power
    power_checked
 
+Trigonometric Functions
+-----------------------
+
+Trigonometric functions are also supported, and also offer ``_checked``
+variants which detect domain and range errors where appropriate.
```

Comment:
Are range errors actually detected?

Reply:
They are not, because none of these functions can raise a range error, except in case of underflow, in which case they're defined to return a correctly rounded result, so I'll revise the docs.